backend/core/src/app.module.ts

```diff
@@ -40,7 +40,27 @@ import { CatchAllFilter } from "./shared/filters/catch-all-filter"
 
 export function applicationSetup(app: INestApplication) {
   const { httpAdapter } = app.get(HttpAdapterHost)
-  const allowList = process.env.CORS_ORIGINS || []
+
+  // It isn't usually possible to pass in an array via env var, so we'll turn
+  // a string into an array when needed
+
+  // Default to empty array
+  let allowList = []
+
+  // If CORS_ORIGINS is set...
+  if (process.env.CORS_ORIGINS) {
+    // use it directly if it's an array
+    if (Array.isArray(process.env.CORS_ORIGINS)) {
+      allowList = process.env.CORS_ORIGINS
+      // otherwise split on "," if it's a string
+    } else if (typeof process.env.CORS_ORIGINS == "string") {
+      allowList = process.env.CORS_ORIGINS.split(",")
+    }
+  }
+
+  console.log("Accepting cross origin requests from:")
```

Comment:
I only added these log statements because it would make testing easier for reviewers than having to set up breakpoints.  Unless anyone objects, I'll take them out again prior to merge to avoid adding noise

Reply:
sgtm